Automattic/sensei
# Fix deprecated argument being used by `get_user_question_answer` and `get_user_question_grade` methods

Fixes #2980

#### Changes proposed in this Pull Request:

* Update the `get_user_question_answer` and `get_user_question_grade` handling calls without `user_id`.

#### Testing instructions (Copied from the issue):

* Create a course with a free preview lesson.
* Add a quiz with at least one question to the lesson.
* As a logged out user, browse directly to the quiz (e.g. your-site/quiz/your-lesson).
* Check the logs. Should not have deprecation notices.

Also, you can test with a logged in to confirm it's working properly.
- Log in
- Answer the quiz and check if everything works properly.
